vignettes/advantages.Rmd

```diff
@@ -0,0 +1,176 @@
+---
+title: "The advantages of tic"
+author: "Patrick Schratz, Kirill Müller"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Kirill Müller}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+So, what's the advantage of using *tic* over the standard CI process?
+
+1. Deployment to a Git repository is greatly simplified. 
+   Update your repository with results from the CI build.
+
+1. Support for R packages and other kinds of project (bookdown, blogdown, etc.), with predefined templates. 
+   Set up your project to deploy rendered versions of your book or blog with a single push to Git.
+
+1. Workflow specification in a single `.R` file, regardless of CI system used.  
+   Forget about `.yml` files or web browser configurations.
+   
+Other minor advantages include the use of `rcmdcheck::rcmdcheck()` for package checking a and robust caching approach of dependencies.
+
+## Easier deployment
+
+You may know that CI services can be used to automatically deploy files. 
+This means that these services can push certain files created during the build to repositories (Github, Gitlab, etc.).
+A common use case is to build your `pkgdown` site and conditionally push changes to your repo.
+This practice saves you from building the whole `pkgdown` site locally and then committing manually all the changed files - either Travis or Appveyor will do it for you! 
+*tic* takes care of all the setup work (setting up SSH keys, permission granting for the repo on Github and the CI services).
```

Comment:
Maybe bullet points give us more space here? We could also mention setup of PATs.

(permission granting -> granting permission)

Reply:
https://github.com/ropenscilabs/tic/pull/80/commits/d0806a8119e97023ab50c0d5027f064efb3d4836